docs/examples/auto-instrumentation/client.py

```diff
@@ -31,16 +30,13 @@
     BatchSpanProcessor(ConsoleSpanExporter())
 )
 
-
-assert len(argv) == 2
-
 with tracer.start_as_current_span("client"):
     with tracer.start_as_current_span("client-server"):
         headers = {}
         inject(headers)
         requested = get(
             "http://localhost:8082/server_request",
-            params={"param": argv[1]},
+            params={"param": "testing"},
```

Comment:
I suppose we could allow a param and fall back to "testing", keeping backward compatibility. On the other hand, this is just a tiny test script so the change as-is is probably fine.